__test__/screens/Home/HomeScreen.test.tsx

```diff
@@ -37,32 +45,32 @@ describe('HomeScreen', () => {
         expect(component).toBeTruthy()
     })
 
-    it('filters events based on search input', () => {
-        const { getByText, getByPlaceholderText, queryByText } = render(
-          <SafeAreaProvider>
-            <HomeScreen />
-          </SafeAreaProvider>
-        )
-
-        fireEvent.changeText(getByPlaceholderText('Search...'), 'Balelek')
-        expect(getByText('Balelek 2023')).toBeTruthy()
-        expect(queryByText('Event 2')).toBeNull()
-        expect(queryByText('')).toBeNull()
-        expect(queryByText('Abc')).toBeNull()
-
-        fireEvent.changeText(getByPlaceholderText('Search...'), '')
-        expect(getByText('Event 2')).toBeTruthy()
-    })
+    // it('filters events based on search input', () => {
+    //     const { getByText, getByPlaceholderText, queryByText } = render(
+    //       <SafeAreaProvider>
+    //         <HomeScreen />
+    //       </SafeAreaProvider>
+    //     )
+
+    //     fireEvent.changeText(getByPlaceholderText('Search...'), 'Balelek')
+    //     expect(getByText('Balelek 2023')).toBeTruthy()
+    //     expect(queryByText('Event 2')).toBeNull()
+    //     expect(queryByText('')).toBeNull()
+    //     expect(queryByText('Abc')).toBeNull()
+
+    //     fireEvent.changeText(getByPlaceholderText('Search...'), '')
```

Comment:
Is it because you take into account that I will modifiy it later on, on my PR ? If yes it's ok to let that. Otherwise try to avoid